Cargo.toml

```diff
@@ -17,9 +17,13 @@ to Ethereum smart contracts.
 name = "ethcontract"
 
 [features]
-default = ["derive"]
+default = ["derive", "http", "http-tls", "ws", "ws-tls"]
```

Comment:
I also tried not changing the default features but then setting up the examples becomes more difficult.
Most examples need "http" but I haven't found a good way to enable an ethcontract feature for the examples. If these features aren't default then the examples need to be run like `cargo run --example abi --feature http`.
tokio does it by having the examples by it's own crate that can depend on `../` with all features enabled.

Reply:
I think its fine to have these enabled by default. I think cargo does not allow you to specify features for examples only, you can however specify `required-features` so that the example doesn't run unless the features are specified on the command line, but this becomes combersome.